.gitignore

```diff
@@ -37,6 +37,7 @@ backend/mail/
 backend/log/*.log*
 backend/log/nginx/*.log*
 .env
+!/frontend/.env
```

Comment:
Can't this be put into the frontend .gitignore?

Reply:
yes, I don't know why I didn't try that first!